api/tests/repositories/planets_repository_test.py

```diff
@@ -0,0 +1,41 @@
+from src.models.anomalies_models import Anomaly
+from src.models.planets_model import Planet
+from src.repositories.anomaly_repository import AnomalyRepository
+from src.repositories.planet_repository import PlanetRepository
+from tests.fixtures.anomalies_factory import AnomalyFactory
+from tests.fixtures.media_factory import MediaFactory
+from tests.fixtures.planet_factory import PlanetFactory
+
+
+class TestCreatePlanet:
+    def test_create_planet(self, client, db_session):
+        planet = Planet(
+            name="planet_exemple",
+            description="just an exemple of planet",
+            url="to_define",
+        )
+        planet_repository = PlanetRepository(db_session)
+        response = planet_repository.save(planet)
+        assert response.name == planet.name
+        assert response.type == planet.type
+        assert response.description == planet.description
+        assert response.url == planet.url
```

Comment:
Renseigne toi sur le AAA (Arrange, Act, Assert)

```suggestion
        # Arrange
        planet = Planet(
            name="planet_exemple",
            description="just an exemple of planet",
            url="to_define",
        )
        planet_repository = PlanetRepository(db_session)
        
        # Act
        response = planet_repository.save(planet)
        
        # Assert
        assert response.name == planet.name
        assert response.type == planet.type
        assert response.description == planet.description
        assert response.url == planet.url
```